packages/flutter/lib/src/rendering/box.dart

```diff
@@ -1341,6 +1341,10 @@ class _IntrinsicDimensionsCacheEntry {
 /// [computeMinIntrinsicWidth], [computeMaxIntrinsicWidth],
 /// [computeMinIntrinsicHeight], [computeMaxIntrinsicHeight].
 ///
+/// Be sure to set [debugCheckIntrinsicSizes] to true in your unit tests if you
+/// do override any of these methods, which will add additional checks to
+/// help validate your implementation.
+///
```

Comment:
we might want to put this on the docs for the computeIntrinsic* methods too, come to think of it

Reply:
Done!